docs/architecture/shared-nothing.txt

```diff
@@ -99,8 +99,8 @@ operating on a shard level.
 Multi Node Setup: Clusters
 ==========================
 
-A CrateDB cluster is a set of two or more instances of CrateDB running on
-different hosts forming a single database unit. For inter-node communication
+A CrateDB cluster is a set of two or more instances of CrateDB referred as nodes
+running on different hosts forming a single database unit. For inter-node communication
```

Comment:
This sentence is convoluted. I suggest:

"A CrateDB cluster is a set of two or more CrateDB instances, referred to as nodes, running on different hosts which form a single database unit"